starport/cmd/validation.go

```diff
@@ -0,0 +1,28 @@
+package starportcmd
+
+import (
+	"context"
+
+	"github.com/spf13/cobra"
+	"github.com/tendermint/starport/starport/pkg/placeholder"
+)
+
+// FlagNoValidation disables validation.
+const FlagNoValidation = "no-validation"
+
+// RegisterValidationFlags adds validation specific flags to cobra.Command.
+func RegisterValidationFlags(cmd *cobra.Command) {
+	cmd.Flags().Bool(FlagNoValidation, false, "Disable validation.")
```

Comment:
Same, I would change the description to "generate files for the component without modifying existing ones"